frontend/src/components/search/SearchBar.tsx

```diff
@@ -24,7 +24,7 @@ export default function SearchBar({ initialQueryParams }: SearchBarProps) {
         className="usa-input maxw-none"
         id="search-field"
```

Comment:
Should this match the `name`? I don't think we're using the `search-field` ID anywhere

Reply:
Yea good catch, we need to update the `htmlFor` to be the same as the id for accessibility reasons - so let's have the htmlFor, id, and name all be `query` (which matches the query params)